Composer/packages/extensions/plugin-loader/src/composerPluginRegistration.ts

```diff
@@ -88,6 +88,20 @@ export class ComposerPluginRegistration {
     this.loader.extensions.runtimeTemplates.push(plugin);
   }
 
+  /**************************************************************************************
+   * Add Bot Template (aka, SampleBot)
+   *************************************************************************************/
+  public addBotTemplate(template: BotTemplate) {
+    this.loader.extensions.botTemplates.push(template);
+  }
+
+  /**************************************************************************************
+   * Add Base Template (aka, BoilerPlate)
+   *************************************************************************************/
+  public addBaseTemplate(template: BotTemplate) {
```

Comment:
I don't really understand this one. What is meant by boilerplate?

Reply:
Thanks, Andy, boilerPlate is introduced by https://github.com/microsoft/BotFramework-Composer/pull/2837 which is work as a `baseTemplate` for all sample bot. For example, contains as a README.md just now.

This PR really don't add any features yet, just refactor the structure.